### DISCUSSION TITLE
Can't create the pull request

### DISCUSSION THREAD

**Discussion by Walter-Obrien** [chiche-ds/30-Day-Terraform-challenge-#24] [Q&A]
After Committing the changes to my new branch I can't create a pull request. I have the message "There isn’t anything to compare" and I don't know how to go about it

**Comment by Elias-hub-bash** [comment]
I have the same response. I am sure because it has nothing in fact to campare since you gave your details that are not same with what is in Chiche's.

**Comment by chiche-ds** [comment]
Use git status to check the status of your work to be sure that you made some edits and committed to them. you can also use git log  to see what is going on in your logs

**Comment by Elias-hub-bash** [comment]
Aaaaaaaaaah instead go to your command line (CLI) and type in the following command  **

> git pull origin master

(or main, depending on your setup)
and press enter

you will see 

From https://github.com/Elias-hub-bash/30-Day-Terraform-challenge-
 * branch            master     -> FETCH_HEAD
Already up to date.

**Comment by Gaturu** [comment]
@chiche-ds I have summited my pull request but unable to see my .md file on the daily task. I have done for Day 2 and Day 3

**Reply by chiche-ds** [reply]
it took some time to merge all the pull requests. check and confirm with me if you still can see your daily.md file
